Some branch

cardosov opened the pull request:
testing purposes

cardosov commented:
<color style='color:red'>Tests failed</span>:
**1** test(s) failed in a total of **1109**. **0** were skipped!

  **org.pentaho.metaverse.api.PropertiesHolderTest.testSetProperty**|
-------|
  ```expected:<myValue> but was:<null>```|

  **org.pentaho.metaverse.api.PropertiesHolderTest.testSetProperty**|
-------|
  ```expected:<myValue> but was:<null>```|

buildguy commented:
## Failed Tests
#### 1 test(s) failed in a total of 1109. 0 were skipped![details](http://ciren.pentaho.com/job/misc/job/vitors-stuff/job/multi-prs/job/PR-2/9/testReport)

**org.pentaho.metaverse.api.PropertiesHolderTest.testSetProperty**|
-------------|
```expected:<myValue> but was:<null>```|

buildguy commented:
## :warning: Failed Tests
####  :cloud_with_rain: 1 test(s) failed in a total of 1109. 0 were skipped! [details](http://ciren.pentaho.com/job/misc/job/vitors-stuff/job/multi-prs/job/PR-2/10/testReport)

:small_red_triangle_down: **org.pentaho.metaverse.api.PropertiesHolderTest.testSetProperty**|
-------------|
```expected:<myValue> but was:<null>```|

buildguy commented:
## :warning: Failed Tests
####  :cloud_with_rain: 2 test(s) failed in a total of 1109. 0 were skipped! [details](http://ciren.pentaho.com/job/misc/job/vitors-stuff/job/multi-prs/job/PR-2/11/testReport)

:small_red_triangle_down: **org.pentaho.metaverse.impl.MetaverseBuilderTest.testDeleteNode**|
-------------|
```Node was not added as a Vertex in the graph```|
:small_red_triangle_down: **org.pentaho.metaverse.api.PropertiesHolderTest.testSetProperty**|
-------------|
```expected:<myValue> but was:<null>```|

buildguy commented:
## :warning: Failed Tests
####  :cloud_with_rain: 2 test(s) failed in a total of 1109. 0 were skipped! [details](http://ciren.pentaho.com/job/misc/job/vitors-stuff/job/multi-prs/job/PR-2/12/testReport)

:small_red_triangle_down: **org.pentaho.metaverse.impl.MetaverseBuilderTest.testDeleteNode**|
-------------|
```Node was not added as a Vertex in the graph```|

:small_red_triangle_down: **org.pentaho.metaverse.api.PropertiesHolderTest.testSetProperty**|
-------------|
```expected:<myValue> but was:<null>```|

buildguy commented:
## :warning: Failed Tests
####  :cloud_with_rain: 2 test(s) failed in a total of 1109. 0 were skipped! [details](http://ciren.pentaho.com/job/misc/job/vitors-stuff/job/multi-prs/job/PR-2/13/testReport)

:small_red_triangle_down: **org.pentaho.metaverse.impl.MetaverseBuilderTest.testDeleteNode**|
-------------|
```Node was not added as a Vertex in the graph```|
:small_red_triangle_down: **org.pentaho.metaverse.api.PropertiesHolderTest.testSetProperty**|
```expected:<myValue> but was:<null>```|

buildguy commented:
## :warning: Failed Tests####  :cloud_with_rain: 2 test(s) failed in a total of 1109. 0 were skipped! [details](http://ciren.pentaho.com/job/misc/job/vitors-stuff/job/multi-prs/job/PR-2/14/testReport)

:small_red_triangle_down: **org.pentaho.metaverse.impl.MetaverseBuilderTest.testDeleteNode**|
-------------|
```Node was not added as a Vertex in the graph```|
:small_red_triangle_down: **org.pentaho.metaverse.api.PropertiesHolderTest.testSetProperty**|
```expected:<myValue> but was:<null>```|

buildguy commented:
## :warning: Failed Tests
:cloud_with_rain: 2 test(s) failed in a total of 1109. 0 were skipped! [details](http://ciren.pentaho.com/job/misc/job/vitors-stuff/job/multi-prs/job/PR-2/15/testReport)|-------------|
:small_red_triangle_down: **org.pentaho.metaverse.impl.MetaverseBuilderTest.testDeleteNode**|
```Node was not added as a Vertex in the graph```|
:small_red_triangle_down: **org.pentaho.metaverse.api.PropertiesHolderTest.testSetProperty**|
```expected:<myValue> but was:<null>```|

buildguy commented:
## :warning: Failed Tests
:cloud_with_rain: 2 test(s) failed in a total of 1109. 0 were skipped! [details](http://ciren.pentaho.com/job/misc/job/vitors-stuff/job/multi-prs/job/PR-2/16/testReport)|
-------------|
:small_red_triangle_down: **org.pentaho.metaverse.impl.MetaverseBuilderTest.testDeleteNode**|
```Node was not added as a Vertex in the graph```|
:small_red_triangle_down: **org.pentaho.metaverse.api.PropertiesHolderTest.testSetProperty**|
```expected:<myValue> but was:<null>```|

buildguy commented:
## :warning: Failed Tests
. :cloud_with_rain: 2 test(s) failed in a total of 1109. 0 were skipped! [details](http://ciren.pentaho.com/job/misc/job/vitors-stuff/job/multi-prs/job/PR-2/17/testReport)|
├── :small_red_triangle_down: **org.pentaho.metaverse.impl.MetaverseBuilderTest.testDeleteNode**
│   ├── ```Node was not added as a Vertex in the graph```
├── :small_red_triangle_down: **org.pentaho.metaverse.api.PropertiesHolderTest.testSetProperty**
│   ├── ```expected:<myValue> but was:<null>```

buildguy commented:
### :warning: Failed Tests
---
##### :cloud_with_lightning_and_rain: 2 test(s) failed:
- **org.pentaho.metaverse.impl.MetaverseBuilderTest.testDeleteNode**
  - ```Node was not added as a Vertex in the graph```
- **org.pentaho.metaverse.api.PropertiesHolderTest.testSetProperty**
  - ```expected:<myValue> but was:<null>```
---
|Total|Failed|Skipped|
|---|---|---|
|1109|2|0|
|[Details](http://ciren.pentaho.com/job/misc/job/vitors-stuff/job/multi-prs/job/PR-2/19/testReport)|
|----|

buildguy commented:
### :warning: Failed Tests
---
#### :cloud_with_lightning_and_rain: 2 test(s) failed:
- **org.pentaho.metaverse.impl.MetaverseBuilderTest.testDeleteNode**
  - ```Node was not added as a Vertex in the graph```
- **org.pentaho.metaverse.api.PropertiesHolderTest.testSetProperty**
  - ```expected:<myValue> but was:<null>```
---
|Total|Failed|Skipped|
|---|---|---|
|1109|2|0|

|[Details](http://ciren.pentaho.com/job/misc/job/vitors-stuff/job/multi-prs/job/PR-2/20/testReport)|
|----|

buildguy commented:
### :warning: Failed Tests
---
#### :cloud_with_lightning_and_rain: 2 test(s) failed:
- **org.pentaho.metaverse.impl.MetaverseBuilderTest.testDeleteNode**
  - ```Node was not added as a Vertex in the graph```
- **org.pentaho.metaverse.api.PropertiesHolderTest.testSetProperty**
  - ```expected:<myValue> but was:<null>```
---
|Total|Failed|Skipped|
|---|---|---|
|1109|2|0|

|[Details](http://ciren.pentaho.com/job/misc/job/vitors-stuff/job/multi-prs/job/PR-2/21/testReport)|
|----|

buildguy commented:
### :warning: Failed Tests
---
#### :cloud_with_lightning_and_rain: 2 test(s) failed:
- **org.pentaho.metaverse.impl.MetaverseBuilderTest.testDeleteNode**
  - ```Node was not added as a Vertex in the graph```
- **org.pentaho.metaverse.api.PropertiesHolderTest.testSetProperty**
  - ```expected:<myValue> but was:<null>```
---
|Total|Failed|Skipped|
|---|---|---|
|1109|2|0|

|[Details](http://ciren.pentaho.com/job/misc/job/vitors-stuff/job/multi-prs/job/PR-2/22/testReport)|
|----|

buildguy commented:
### :warning: Failed Tests
---
#### :cloud_with_lightning_and_rain: 2 test(s) failed:
- **org.pentaho.metaverse.impl.MetaverseBuilderTest.testDeleteNode**
  - ```Node was not added as a Vertex in the graph```
- **org.pentaho.metaverse.api.PropertiesHolderTest.testSetProperty**
  - ```expected:<myValue> but was:<null>```
---
|Total|Failed|Skipped|
|---|---|---|
|1109|2|0|

|[Details](http://ciren.pentaho.com/job/misc/job/vitors-stuff/job/multi-prs/job/PR-2/23/testReport)|
|----|

cardosov commented:
### :warning: Failed Tests
---
#### :cloud_with_lightning_and_rain: 2 test(s) failed:

- **org.pentaho.metaverse.api.PropertiesHolderTest.testSetProperty**
  - ```expected:<myValue> but was:<null>```
- **org.pentaho.metaverse.impl.MetaverseBuilderTest.testDeleteNode**
  - ```Node was not added as a Vertex in the graph```
---

|Total|Failed|Skipped|
|---|---|---|
|1109|2|0|

|[Details](http://0.0.0.0:8080/job/pipeline-jobs/job/pentaho-metaverse/job/PR-2/6//testReport)|
|----|

cardosov commented:
### :warning: Failed Tests
---
#### :cloud_with_lightning_and_rain: 2 test(s) failed:

- **org.pentaho.metaverse.api.PropertiesHolderTest.testSetProperty**
  - ```expected:<myValue> but was:<null>```
- **org.pentaho.metaverse.impl.MetaverseBuilderTest.testDeleteNode**
  - ```Node was not added as a Vertex in the graph```
---

|Total|Failed|Skipped|
|---|---|---|
|1109|2|0|

|[Details](http://0.0.0.0:8080/job/pipeline-jobs/job/pentaho-metaverse/job/PR-2/7/testReport)|
|----|

cardosov commented:
### :warning: Failed Tests
---
#### :cloud_with_lightning_and_rain: 2 test(s) failed:

- **org.pentaho.metaverse.api.PropertiesHolderTest.testSetProperty**
  - ```expected:<myValue> but was:<null>```
- **org.pentaho.metaverse.impl.MetaverseBuilderTest.testDeleteNode**
  - ```Node was not added as a Vertex in the graph```
---

|Total|Failed|Skipped|
|---|---|---|
|1109|2|0|

|[Details](http://0.0.0.0:8080/job/pipeline-jobs/job/pentaho-metaverse/job/PR-2/8/testReport)|
|----|

cardosov commented:
### :warning: Failed Tests
---
#### :cloud_with_lightning_and_rain: 2 test(s) failed:

- **org.pentaho.metaverse.api.PropertiesHolderTest.testSetProperty**
  - ```expected:<myValue> but was:<null>```
- **org.pentaho.metaverse.impl.MetaverseBuilderTest.testDeleteNode**
  - ```Node was not added as a Vertex in the graph```
---

|Total|Failed|Skipped|
|---|---|---|
|1109|2|0|

|[Details](http://0.0.0.0:8080/job/pipeline-jobs/job/pentaho-metaverse/job/PR-2/9/testReport)|
|----|

cardosov commented:
### :warning: Failed Tests
---
#### :cloud_with_lightning_and_rain: 2 test(s) failed:

- **org.pentaho.metaverse.api.PropertiesHolderTest.testSetProperty**
  - ```expected:<myValue> but was:<null>```
- **org.pentaho.metaverse.impl.MetaverseBuilderTest.testDeleteNode**
  - ```Node was not added as a Vertex in the graph```
---

|Total|Failed|Skipped|
|---|---|---|
|1109|2|0|

|[Details](http://0.0.0.0:8080/job/pipeline-jobs/job/pentaho-metaverse/job/PR-2/12/testReport)|
|----|

cardosov commented:
### :warning: Failed Tests
---
#### :cloud_with_lightning_and_rain: 2 test(s) failed:

- **org.pentaho.metaverse.api.PropertiesHolderTest.testSetProperty**
  - ```expected:<myValue> but was:<null>```
- **org.pentaho.metaverse.impl.MetaverseBuilderTest.testDeleteNode**
  - ```Node was not added as a Vertex in the graph```
---

|Total|Failed|Skipped|
|---|---|---|
|1109|2|0|

|[Details](http://0.0.0.0:8080/job/pipeline-jobs/job/pentaho-metaverse/job/PR-2/1/testReport)|
|----|

cardosov commented:
### :warning: Failed Tests
---
#### :cloud_with_lightning_and_rain: 2 test(s) failed:

- **org.pentaho.metaverse.api.PropertiesHolderTest.testSetProperty**
  - ```expected:<myValue> but was:<null>```
- **org.pentaho.metaverse.impl.MetaverseBuilderTest.testDeleteNode**
  - ```Node was not added as a Vertex in the graph```
---

|Total|Failed|Skipped|
|---|---|---|
|1109|2|0|

|[Details](http://0.0.0.0:8080/job/pipeline-jobs/job/pentaho-metaverse/job/PR-2/2/testReport)|
|----|

cardosov commented:
### :warning: Failed Tests
---
#### :cloud_with_lightning_and_rain: 2 test(s) failed:

- **org.pentaho.metaverse.api.PropertiesHolderTest.testSetProperty**
  - ```expected:<myValue> but was:<null>```
- **org.pentaho.metaverse.impl.MetaverseBuilderTest.testDeleteNode**
  - ```Node was not added as a Vertex in the graph```
---

|Total|Failed|Skipped|
|---|---|---|
|1109|2|0|

|[Details](http://0.0.0.0:8080/job/pipeline-jobs/job/pentaho-metaverse/job/PR-2/3/testReport)|
|----|

cardosov commented:
### :warning: Failed Tests
---
#### :cloud_with_lightning_and_rain: 2 test(s) failed:

- **org.pentaho.metaverse.api.PropertiesHolderTest.testSetProperty**
  - ```expected:<myValue> but was:<null>```
- **org.pentaho.metaverse.impl.MetaverseBuilderTest.testDeleteNode**
  - ```Node was not added as a Vertex in the graph```
---

|Total|Failed|Skipped|
|---|---|---|
|1109|2|0|

|[Details](http://0.0.0.0:8080/job/pipeline-jobs/job/pentaho-metaverse/job/PR-2/1/testReport)|
|----|

cardosov commented:
### :warning: Failed Tests
---
#### :cloud_with_lightning_and_rain: 2 test(s) failed:

- **org.pentaho.metaverse.api.PropertiesHolderTest.testSetProperty**
  - ```expected:<myValue> but was:<null>```
- **org.pentaho.metaverse.impl.MetaverseBuilderTest.testDeleteNode**
  - ```Node was not added as a Vertex in the graph```
---

|Total|Failed|Skipped|
|---|---|---|
|1109|2|0|

|[Details](http://0.0.0.0:8080/job/pipeline-jobs/job/pentaho-metaverse/job/PR-2/2/testReport)|
|----|

cardosov commented:
### :warning: Failed Tests
---
#### :cloud_with_lightning_and_rain: 2 test(s) failed:

- **org.pentaho.metaverse.api.PropertiesHolderTest.testSetProperty**
  - ```expected:<myValue> but was:<null>```
- **org.pentaho.metaverse.impl.MetaverseBuilderTest.testDeleteNode**
  - ```Node was not added as a Vertex in the graph```
---

|Total|Failed|Skipped|
|---|---|---|
|1109|2|0|

|[Details](http://0.0.0.0:8080/job/pipeline-jobs/job/pentaho-metaverse/job/PR-2/1/testReport)|
|----|

cardosov commented:
something

---
#### :cloud_with_lightning_and_rain: 1 test(s) failed:

- **org.hitachivantara.ci.TestFileUtils.test the creation of symbolic links**
  - ```Condition not satisfied:

Files.isSymbolicLink(result) && Files.isReadable(result) == readable
|     |              |       |  |     |          |       |  |
|     true           |       |  |     true       |       |  false
|                    |       |  |                |       false
|                    |       |  |                target/something-1559320117942/srcAtTargetLink
|                    |       |  class java.nio.file.Files
|                    |       false
|                    target/something-1559320117942/srcAtTargetLink
class java.nio.file.Files
```
---

buildguy commented:
### :warning: Failed Tests
---
#### :cloud_with_lightning_and_rain: 2 test(s) failed:

- **org.pentaho.metaverse.api.PropertiesHolderTest.testSetProperty**
```
expected:<myValue> but was:<null>
```
- **org.pentaho.metaverse.impl.MetaverseBuilderTest.testDeleteNode**
```
Node was not added as a Vertex in the graph
```
---

|Total|Failed|Skipped|
|---|---|---|
|1109|2|0|

|[Details](http://ciren.pentaho.net/job/misc/job/vitors-stuff/job/velociraptor-jobs/job/cardosov-metaverse/job/PR-2/1/testReport)|
|----|

buildguy commented:
### :warning: Failed Tests
---
#### :cloud_with_lightning_and_rain: 2 test(s) failed:

- **org.pentaho.metaverse.api.PropertiesHolderTest.testSetProperty**
```
expected:<myValue> but was:<null>
```
- **org.pentaho.metaverse.impl.MetaverseBuilderTest.testDeleteNode**
```
Node was not added as a Vertex in the graph
```
---

|Total|Failed|Skipped|
|---|---|---|
|1109|2|0|

|[Details](http://ciren.pentaho.net/job/misc/job/vitors-stuff/job/velociraptor-jobs/job/cardosov-metaverse/job/PR-2/1/testReport)|
|----|

cardosov commented:
:robot: Pull Request build was aborted
----
Hello,

The build for this Pull Request was aborted. See below the reason(s):
```
Source fork/repository no longer exists! Please consider closing this Pull Request.
```

:information_source:  _This is an automatic message_

cardosov commented:
### :gear:  Command executed:
----
```
mvn clean install
```
### :timer_clock: Timings
----
```
[Configure] (1s)
[Checkout]  (3s)
[Build]     (51s)
   jenkins-shared-libraries : 51s
[Audit]     (21s)
   jenkins-shared-libraries: SonarQube : 20s
```

cardosov commented:
#[----
```
mvn clean verify -B -e -f pom.xml -Daether.connector.resumeDownloads=false -amd -Daudit -P !assemblies -U -pl api,core
```
#[----
```
['Configure':[GENERAL:[746]], 'Checkout':[JOB:[{
  jobID: "pentaho-metaverse",
  archivable: false,
  asynchronous: false,
  atomicScmCheckout: false,
  buildFile: "pom.xml",
  buildFramework: BuildFramework.MAVEN,
  buildWorkDir: "/build2/ciren/workspace/misc/vitors-stuff/backlog-29772-orchestrator/builds/cardosov.pentaho-metaverse.PR-2",
  changed: true,
  checkoutDir: "/build2/ciren/workspace/misc/vitors-stuff/backlog-29772-orchestrator/builds/cardosov.pentaho-metaverse.PR-2",
  directives: "clean install -Daudit -U",
  dockerImage: null,
  execType: JobItem.ExecutionType.FORCE,
  jobGroup: "wingman",
  parallelize: false,
  passOnBuildParameters: true,
  prDirectives: "clean verify -B -e -f pom.xml -Daether.connector.resumeDownloads=false -amd -Daudit -P !assemblies -U -pl api,core",
  prExecType: "AUTO",
  prMerge: false,
  prReportStatus: false,
  prScan: true,
  prStatusLabel: "Wingman Build",
  properties: null,
  root: "",
  scmBranch: "PR-2",
  scmCredentials: "cardosov at github",
  scmID: "cardosov.pentaho-metaverse.PR-2",
  scmInfo: {
    scm: "git",
    connection: JobItem.SCMConnectionType.HTTP,
    organization: "cardosov",
    repository: "pentaho-metaverse",
    checkout: {
      GIT_BRANCH: "origin/PR-2",
      GIT_COMMIT: "e758ab1f5cff063e27b008da2ea7c319b64fc647",
      GIT_PREVIOUS_COMMIT: "origin/master",
      GIT_URL: "https://github.com/cardosov/pentaho-metaverse.git",
      GIT_REV_LIST: [
        "e758ab1f5cff063e27b008da2ea7c319b64fc647", 
        "699978c19199db84f44dca8f1f7c1c51ce2609f4", 
        "9f9ceeb6268df3725c9489bca306027020fbfa4c"
      ]
    },
    changeLog: [
      "core/src/test/java/org/pentaho/metaverse/impl/MetaverseBuilderTest.java", 
      "api/src/test/java/org/pentaho/metaverse/api/PropertiesHolderTest.java", 
      "README.md"
    ]
  },
  scmPoll: true,
  scmRevision: null,
  scmScanInterval: "10",
  scmUrl: "https://github.com/cardosov/pentaho-metaverse.git",
  script: null,
  securityScannable: false,
  settingsFile: null,
  skip: false,
  targetJobName: null,
  testable: true,
  testsArchivePattern: "**/bin/**/TEST*.xml, **/target/**/TEST*.xml, **/build/**/*Test.xml, **/target/**/test*.xml",
  timeout: null,
  versionProperty: null
}:6176], GENERAL:[6221]], 'Build':[JOB:[{
  jobID: "pentaho-metaverse",
  archivable: false,
  asynchronous: false,
  atomicScmCheckout: false,
  buildFile: "pom.xml",
  buildFramework: BuildFramework.MAVEN,
  buildWorkDir: "/build2/ciren/workspace/misc/vitors-stuff/backlog-29772-orchestrator/builds/cardosov.pentaho-metaverse.PR-2",
  changed: true,
  checkoutDir: "/build2/ciren/workspace/misc/vitors-stuff/backlog-29772-orchestrator/builds/cardosov.pentaho-metaverse.PR-2",
  directives: "clean install -Daudit -U",
  dockerImage: null,
  execType: JobItem.ExecutionType.FORCE,
  jobGroup: "wingman",
  parallelize: false,
  passOnBuildParameters: true,
  prDirectives: "clean verify -B -e -f pom.xml -Daether.connector.resumeDownloads=false -amd -Daudit -P !assemblies -U -pl api,core",
  prExecType: "AUTO",
  prMerge: false,
  prReportStatus: false,
  prScan: true,
  prStatusLabel: "Wingman Build",
  properties: null,
  root: "",
  scmBranch: "PR-2",
  scmCredentials: "cardosov at github",
  scmID: "cardosov.pentaho-metaverse.PR-2",
  scmInfo: {
    scm: "git",
    connection: JobItem.SCMConnectionType.HTTP,
    organization: "cardosov",
    repository: "pentaho-metaverse",
    checkout: {
      GIT_BRANCH: "origin/PR-2",
      GIT_COMMIT: "e758ab1f5cff063e27b008da2ea7c319b64fc647",
      GIT_PREVIOUS_COMMIT: "origin/master",
      GIT_URL: "https://github.com/cardosov/pentaho-metaverse.git",
      GIT_REV_LIST: [
        "e758ab1f5cff063e27b008da2ea7c319b64fc647", 
        "699978c19199db84f44dca8f1f7c1c51ce2609f4", 
        "9f9ceeb6268df3725c9489bca306027020fbfa4c"
      ]
    },
    changeLog: [
      "core/src/test/java/org/pentaho/metaverse/impl/MetaverseBuilderTest.java", 
      "api/src/test/java/org/pentaho/metaverse/api/PropertiesHolderTest.java", 
      "README.md"
    ]
  },
  scmPoll: true,
  scmRevision: null,
  scmScanInterval: "10",
  scmUrl: "https://github.com/cardosov/pentaho-metaverse.git",
  script: null,
  securityScannable: false,
  settingsFile: null,
  skip: false,
  targetJobName: null,
  testable: true,
  testsArchivePattern: "**/bin/**/TEST*.xml, **/target/**/TEST*.xml, **/build/**/*Test.xml, **/target/**/test*.xml",
  timeout: null,
  versionProperty: null
}:70843], GENERAL:[73171]], 'Audit':[GENERAL:[20]]]
```
### :warning: Failed Tests
---
#### :cloud_with_lightning_and_rain: 2 test(s) failed:

- **org.pentaho.metaverse.api.PropertiesHolderTest.testSetProperty**
```
expected:<myValue> but was:<null>
```
- **org.pentaho.metaverse.impl.MetaverseBuilderTest.testDeleteNode**
```
Node was not added as a Vertex in the graph
```
---

|Total|Failed|Skipped|
|---|---|---|
|1109|2|0|

|[Details](http://ciren.pentaho.net/job/misc/job/vitors-stuff/job/backlog-29772-orchestrator-jobs/job/pentaho-metaverse/job/PR-2/5/testReport)|
|----|
:information_source: This is an automatic message

cardosov commented:
#[```
mvn clean verify -B -e -f pom.xml -Daether.connector.resumeDownloads=false -amd -Daudit -P !assemblies -U -pl api,core
```
### :warning: Failed Tests
---
#### :cloud_with_lightning_and_rain: 2 test(s) failed:
    - **org.pentaho.metaverse.api.PropertiesHolderTest.testSetProperty**
    ```
        expected:<myValue> but was:<null>
    ```
    - **org.pentaho.metaverse.impl.MetaverseBuilderTest.testDeleteNode**
    ```
        Node was not added as a Vertex in the graph
    ```
---

|Total|Failed|Skipped|
|---|---|---|
|1109|2|0|

|[Details](http://ciren.pentaho.net/job/misc/job/vitors-stuff/job/backlog-29772-orchestrator-jobs/job/pentaho-metaverse/job/PR-2/7/testReport)|
|----|

:information_source:  _This is an automatic message_

cardosov commented:
### :gear: Build finished in 1 min 13 sec and counting with command:
```
mvn clean verify -B -e -f pom.xml -Daether.connector.resumeDownloads=false -amd -Daudit -P !assemblies -U -pl api,core
```
### :warning: Failed Tests
---
#### :cloud_with_lightning_and_rain: 2 test(s) failed:
    - **org.pentaho.metaverse.api.PropertiesHolderTest.testSetProperty**
    ```
        expected:<myValue> but was:<null>
    ```
    - **org.pentaho.metaverse.impl.MetaverseBuilderTest.testDeleteNode**
    ```
        Node was not added as a Vertex in the graph
    ```
---

|Total|Failed|Skipped|
|---|---|---|
|1109|2|0|

|[Details](http://ciren.pentaho.net/job/misc/job/vitors-stuff/job/backlog-29772-orchestrator-jobs/job/pentaho-metaverse/job/PR-2/8/testReport)|
|----|

:information_source:  _This is an automatic message_

cardosov commented:
### :gear: Build finished in 1 min 5 sec and counting with command:
```
mvn clean verify -B -e -f pom.xml -Daether.connector.resumeDownloads=false -amd -Daudit -P !assemblies -U -pl api,core
```
### :warning: Failed Tests
---
#### :cloud_with_lightning_and_rain: 2 test(s) failed:
- **org.pentaho.metaverse.api.PropertiesHolderTest.testSetProperty**
```
    expected:<myValue> but was:<null>
```
- **org.pentaho.metaverse.impl.MetaverseBuilderTest.testDeleteNode**
```
    Node was not added as a Vertex in the graph
```
---

|Total|Failed|Skipped|
|---|---|---|
|1109|2|0|

|[Details](http://ciren.pentaho.net/job/misc/job/vitors-stuff/job/backlog-29772-orchestrator-jobs/job/pentaho-metaverse/job/PR-2/9/testReport)|
|----|

:information_source:  _This is an automatic message_

cardosov commented:
### :gear: Build finished in 18120 days, 9 hours, 39 minutes, 30.835 seconds with command:
```
mvn clean verify -B -e -f pom.xml -Daether.connector.resumeDownloads=false -amd -Daudit -P !assemblies -U -pl api,core
```
### :warning: Failed Tests
---
#### :cloud_with_lightning_and_rain: 2 test(s) failed:
- **org.pentaho.metaverse.api.PropertiesHolderTest.testSetProperty**
```
    expected:<myValue> but was:<null>
```
- **org.pentaho.metaverse.impl.MetaverseBuilderTest.testDeleteNode**
```
    Node was not added as a Vertex in the graph
```
---

|Total|Failed|Skipped|
|---|---|---|
|1109|2|0|

|[Details](http://ciren.pentaho.net/job/misc/job/vitors-stuff/job/backlog-29772-orchestrator-jobs/job/pentaho-metaverse/job/PR-2/10/testReport)|
|----|

:information_source:  _This is an automatic message_

cardosov commented:
### :gear: Build finished in 1 minutes, 48.506 seconds with command:
```
mvn clean verify -B -e -f pom.xml -Daether.connector.resumeDownloads=false -amd -Daudit -P !assemblies -U -pl api,core
```
### :warning: Failed Tests
---
#### :cloud_with_lightning_and_rain: 2 test(s) failed:
- **org.pentaho.metaverse.api.PropertiesHolderTest.testSetProperty**
```
    expected:<myValue> but was:<null>
```
- **org.pentaho.metaverse.impl.MetaverseBuilderTest.testDeleteNode**
```
    Node was not added as a Vertex in the graph
```
---

|Total|Failed|Skipped|
|---|---|---|
|1109|2|0|

|[Details](http://ciren.pentaho.net/job/misc/job/vitors-stuff/job/backlog-29772-orchestrator-jobs/job/pentaho-metaverse/job/PR-2/11/testReport)|
|----|

:information_source:  _This is an automatic message_

cardosov commented:
### :gear: Build finished in 1 min 23 sec with command:
```
mvn clean verify -B -e -f pom.xml -Daether.connector.resumeDownloads=false -amd -Daudit -P !assemblies -U -pl api,core
```
### :warning: Failed Tests
---
#### :cloud_with_lightning_and_rain: 2 test(s) failed:
- **org.pentaho.metaverse.api.PropertiesHolderTest.testSetProperty**
```
    expected:<myValue> but was:<null>
```
- **org.pentaho.metaverse.impl.MetaverseBuilderTest.testDeleteNode**
```
    Node was not added as a Vertex in the graph
```
---

|Total|Failed|Skipped|
|---|---|---|
|1109|2|0|

|[Details](http://ciren.pentaho.net/job/misc/job/vitors-stuff/job/backlog-29772-orchestrator-jobs/job/pentaho-metaverse/job/PR-2/12/testReport)|
|----|

:information_source:  _This is an automatic message_

cardosov commented:
### :gear: Build finished in 14 sec with command:
```
${command}
```

 :information_source:  _This is an automatic message_

cardosov commented:
### :gear: Build finished in 1 min 0 sec with command:
```
mvn clean verify -B -e -f pom.xml -Daether.connector.resumeDownloads=false -amd -Daudit -P !assemblies -U -pl core
```
:ok_hand: All tests passed!
 ---

|Total|Failed|Skipped|
|---|---|---|
|865|0|0|

|[Details](http://ciren.pentaho.net/job/misc/job/vitors-stuff/job/backlog-29772-orchestrator-jobs/job/pentaho-metaverse/job/PR-2/7/testReport)|
|----|

:information_source:  _This is an automatic message_

cardosov commented:
### :gear: Build finished in 1 min 1 sec with command:
```
mvn clean verify -B -e -f pom.xml -Daether.connector.resumeDownloads=false -amd -Daudit -P !assemblies -U -pl core
```
:ok_hand: All tests passed!
 ---

|Total|Failed|Skipped|
|---|---|---|
|865|0|0|

|[Details](http://ciren.pentaho.net/job/misc/job/vitors-stuff/job/backlog-29772-orchestrator-jobs/job/pentaho-metaverse/job/PR-2/9/testReport)|
|----|

:information_source:  _This is an automatic message_

cardosov commented:
### :gear: Build finished in 1 min 58 sec with command:
```
mvn clean verify -B -e -f pom.xml -Daether.connector.resumeDownloads=false -amd -Daudit -P !assemblies -U -pl core
```
:ok_hand: All tests passed!
 ---

|Total|Failed|Skipped|
|---|---|---|
|865|0|0|

|[Details](http://ciren.pentaho.net/job/misc/job/vitors-stuff/job/backlog-29772-orchestrator-jobs/job/pentaho-metaverse/job/PR-2/11/testReport)|
|----|

:information_source:  _This is an automatic message_

cardosov commented:
### :gear: Build finished in 1 min 20 sec with command:
```
mvn clean verify -B -e -f pom.xml -Daether.connector.resumeDownloads=false -amd -Daudit -P !assemblies -U -pl core
```
:ok_hand: All tests passed!
 ---

|Total|Failed|Skipped|
|---|---|---|
|865|0|0|

|[Details](http://ciren.pentaho.net/job/misc/job/vitors-stuff/job/backlog-29772-orchestrator-jobs/job/pentaho-metaverse/job/PR-2/16/testReport)|
|----|

:information_source:  _This is an automatic message_

cardosov commented:
### :gear: Build finished in 1 min 24 sec with command:
```
mvn clean verify -B -e -f pom.xml -Daether.connector.resumeDownloads=false -amd -Daudit -P !assemblies -U -pl api,core
```
### :warning: Failed Tests
---
#### :cloud_with_lightning_and_rain: 2 test(s) failed:
- **org.pentaho.metaverse.api.PropertiesHolderTest.testSetProperty**
```
    expected:<myValue> but was:<null>
```
- **org.pentaho.metaverse.impl.MetaverseBuilderTest.testDeleteNode**
```
    Node was not added as a Vertex in the graph
```
  ---

|Total|Failed|Skipped|
|---|---|---|
|1109|2|0|

|[Details](http://ciren.pentaho.net/job/misc/job/vitors-stuff/job/backlog-29772-orchestrator-jobs/job/pentaho-metaverse/job/PR-2/17/testReport)|
|----|

:information_source:  _This is an automatic message_
